piety-promoted-v1/modernized/02-piety-promoted-v1.adoc

```diff
@@ -929,9 +975,12 @@ the destruction of the city of London by fire,
 which was six years before it came to pass; and which vision he published before-hand,
 as a warning to the people to repent; part whereof is as follows:
 
-Part of the vision of Humphrey Smith,
+[.embedded-content-document.paper]
+--
+
+_Part of the vision of Humphrey Smith,
```

Comment:
even though this is long, I think it fits more into the category of a letter-heading (instead of just italicizing it). It's kindof a title or heading for what follows.